TrustWalletCore.podspec

```diff
@@ -158,7 +158,7 @@ Pod::Spec.new do |s|
   s.pod_target_xcconfig = {
     'SYSTEM_HEADER_SEARCH_PATHS' => '$(inherited) /usr/local/include'
   }
-  s.prepare_command = 'tools/generate-files && cmake -H. -Bbuild && make -Cbuild protobuf_ext nlohmann_json'
+  s.prepare_command = 'tools/install-dependencies && tools/generate-files'
```

Comment:
can we run pod spec lint for this change somehow?

Reply:
spec lint always fails for me

Reply:
ok, I will try it in trust-core